dev/workflow/hosts/hera.yaml

```
@@ -3,16 +3,16 @@ DMPDIR: '/scratch1/NCEPDEV/global/glopara/dump'
 BASE_GIT: '/scratch1/NCEPDEV/global/glopara/git'
 BASE_DATA: '/scratch1/NCEPDEV/global/glopara/data'
 BASE_IC: '/scratch1/NCEPDEV/global/glopara/data/ICSDIR'
-AERO_INPUTS_DIR: /scratch1/NCEPDEV/global/glopara/data/GEFS_ExtData/20250310
+AERO_INPUTS_DIR: '/scratch1/NCEPDEV/global/glopara/data/GEFS_ExtData/20250310'
 PACKAGEROOT: '/scratch1/NCEPDEV/global/glopara/nwpara'
 HOMEDIR: '/scratch1/NCEPDEV/global/${USER}'
 STMP: '/scratch1/NCEPDEV/stmp2/${USER}'
 PTMP: '/scratch1/NCEPDEV/stmp4/${USER}'
 NOSCRUB: '${HOMEDIR}'
-COMINsyn: '/scratch1/NCEPDEV/global/glopara/com/gfs/prod/syndat'
-COMINecmwf: /scratch1/NCEPDEV/global/glopara/data/external_gempak/ecmwf
-COMINnam: /scratch1/NCEPDEV/global/glopara/data/external_gempak/nam
-COMINukmet: /scratch1/NCEPDEV/global/glopara/data/external_gempak/ukmet
+COM_SYNDAT: '/scratch1/NCEPDEV/global/glopara/com/gfs/prod/syndat'
```

```suggestion
COMINsyndat: '/scratch1/NCEPDEV/global/glopara/com/gfs/prod/syndat'
```
Here and other host yamls

---

Updated, thanks!